### ISSUE TITLE
Screen reader support is lacking.

### ISSUE THREAD

**Issue by dumptruckman** [chenglou/react-radio-group#50]
Please consider making this library better for people with disabilities. I have opened a PR (#47) that will make the very simple changes required to make this widget WAI-ARIA 1.1 compliant.

**Comment by chenglou** [comment]
👍
